File: src/NocWMS-Client/src/app/models/warehouse.ts
import { WarehouseCell } from './warehouse-cell';
export class Warehouse {
    constructor(warehouse?: Partial<Warehouse>) {
        if (warehouse) {
            this.id = warehouse.id;
            this.warehouseName = warehouse.warehouseName;
            this.managerName = warehouse.managerName;
            this.staffId = warehouse.staffId;
            this.province = warehouse.province;
            this.address = warehouse.address;
            this.cells = warehouse.cells;
        }
    }

    public id: number;
    public warehouseName: string;
    public staffId: string;
    public managerName: string;
    public province: string;
    public address: string;
    public cells: WarehouseCell[];
}

File: src/NocWMS-Server/Server.ApiService/Models/Warehouse.cs
using System;
using System.Collections.Generic;
using KiraNet.Camellia.Infrastructure.DomainModel.Data;

namespace Server.ApiService.Models
{
    public class Warehouse : IEntity<int>
    {
        public int Id { get; set; }
        public string WarehouseName { get; set; }
        public string StaffId { get; set; }
        public string Province { get; set; }
        public string Address { get; set; }
        public string Remarks { get; set; }
        public DateTime CreateTime { get; set; }

        public List<WarehouseCell> Cells { get; set; }
        public List<InboundReceipt> InboundReceipts { get; set; }
        public List<OutboundReceipt> OutboundReceipts { get; set; }
        public List<Inventory> Inventories { get; set; }
        public Staff Staff { get; set; }
    }
}